### ISSUE TITLE
Proposal: New content type "Video"

### ISSUE THREAD

**Issue by timrobertson100** [gbif/portal16#58]
_To be considered only after 1st November 2016 when videos are ready for review and consideration_

GBIF are exploring videos as a means to deliver training material and updates.  These will be delivered through YouTube, Vimeo etc.  The generic content type can support this when a video is part of a narrative (e.g. embedded in the middle of an explanation), but there are also likely to be videos that warrant being considered as a top level content type - i.e. a "video" facet type on the sidebar.

I propose that we aim to keep this as simple as possible, with a Video URL a description and keywords / tags, date etc, citation string (to credit those involved in it's production), a license and possible language choices if applicable.  I don't recommend trying to model affiliations with projects, programmes, news, datasets etc. but instead rather rely on the description to capture anything like that.  This will ensure maximum reuse, and easiest implementation.

I imagine a description being along the lines of "This video instructs users how to publish sample based data through an existing installation of an Integrated Publishing Toolkit (IPT).  The video makes use of a [sample file] and describes the best practice for mapping fields in this... etc.  The video was recorded at the first training event of publishers connected with projectX on dd/mm/YYYY and was targeting use of the IPT version 2.3.2"

Videos must be associated with an organisation account, and not be tied to an individual staff member to be suitable for inclusion.  

It could be the case that we use videos embedded in generic content types, but in addition create a specific video content entity as well for the same video.  This way the video would appear inline in some pages and also be a top level discovery item.

@atalavan in particular has created training material which could be repurposed.

**Comment by atalavan** [comment]
When uploading the videos in Youtube, we are already providing a generic description and keywords. We could easily reuse those in the website, if needed.

**Comment by timrobertson100** [comment]
Yes.  We don't want to crawl Youtube, Vimeo etc and it needs to be searchable in GBIF.org so needs entry in the CMS too... It is not unreasonable that people double key it given the limited content.
